src/installer/pkg/sfx/bundle/theme/1028/bundle.wxl

```diff
@@ -1,74 +1,74 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <WixLocalization Culture="en-us" Language="1033" xmlns="http://schemas.microsoft.com/wix/2006/localization">
-  <String Id="Caption">[WixBundleName] 安裝程式</String>
+  <String Id="Caption">[WixBundleName] Installer</String>
   <String Id="Title">[BUNDLEMONIKER]</String>
-  <String Id="Motto">您只需要殼層、文字編輯器和 10 分鐘的時間。
+  <String Id="Motto">You just need a shell, a text editor and 10 minutes of your time.
```

Comment:
looks like all of the installer files got reverted back to English, is that expected?